topic7/src/main/java/controllers/CourseController.java

```diff
@@ -0,0 +1,51 @@
+package controllers;
+
+import models.Course;
+import models.CourseDao;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class CourseController {
+
+	@Autowired
+	private CourseDao _CourseDao;
+
+	@RequestMapping("/create")
+	@ResponseBody
+	public String create(String name) {
+		try {
+			Course Course = new Course(name);
+			_CourseDao.save(Course);
+		} catch (Exception ex) {
+			return "Error creating the user: " + ex.toString();
+		}
+		return "Course succesfully created!";

```

Comment:
A good option in a create method is to return the object just created or a boolean indicating the result